package.json

```diff
@@ -62,6 +62,7 @@
     "request-promise-native": "^1.0.9",
     "semver": "^6.1.1",
     "svgo": "^1.3.2",
+    "terser-webpack-plugin": "^1.4.3",
```

Comment:
I'm not sure this is the correct solution. Essentially it's a transitive dependency listed as a "requires" not a "dependency" which means eslint thinks it may not be there in prod...

Reply:
You're right. I misread the package-lock.

Further googling suggests that it's because we're requiring a transitive dependency in which case this is probably the right fix?

Reply:
agree, it's the right fix!